src/main/java/org/generationcp/middleware/domain/inventory/manager/LotUpdateDto.java

```diff
@@ -0,0 +1,66 @@
+package org.generationcp.middleware.domain.inventory.manager;
+
+
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+@AutoProperty
+public class LotUpdateDto {
```

Comment:
Please move this class as a inner class of LotMultiUpdateRequestDto. It does not make much sense out of that context